examples/device-dashboard/Makefile

```diff
@@ -42,7 +42,6 @@ packed_fs.c: $(wildcard web_root/*) Makefile web_root/main.css web_root/bundle.j
 # Pull and build mbedTLS library. See https://mongoose.ws/tutorials/tls/#how-to-build for TLS build options
 mbedtls:
 	git clone --depth 1 -b v2.28.2 https://github.com/mbed-tls/mbedtls $@
-	$(MAKE) -C mbedtls/library
```

Comment:
This is actually needed to build with a local copy of the library, the third option in Linux/Mac, only one in Windows.
There is no other way to build in Windows, and that is one of the reasons for that section in the tutorial
We tried other ways and this one was the cleanest one running on all three major OSs.